### DISCUSSION TITLE
Middle-click grab scrolling is missing

### DISCUSSION THREAD

**Discussion by thumbwarriordx** [ihhub/fheroes2#8068] [Ideas]
I find this to be something I notice switching between Heroes3 and Fheroes2.

My time with MoBAs embedded grab scroll into my muscle memory by being quicker in certain situations. I like to use it interchangeably with edge scroll.
Just a little thing that would bring the UI better parity with HoMM3

**Comment by ihhub** [comment]
Hi @thumbwarriordx , this can be achieved by left mouse click. fheroes2 is used in many devices and not every single user has a mouse with in wheel.

**Reply by thumbwarriordx** [reply]
Yes but I always forget it's left click instead lol
As I said it's a minor thing but for the sake of muscle memory it could be bound to both left and middle click.

**Comment by oleg-derevenetz** [comment]
Hi @thumbwarriordx there is drag scroll in fheroes2, it just utilizes the left mouse button (not the middle) to work on touch screens in a natural way (since the single finger touch emulates the left mouse button).
